### PR TITLE
Examine Blocks

### PR DESCRIPTION
**Pull request by Geevies**
## About The Pull Request

Ports: https://github.com/tgstation/tgstation/pull/67937

Made some custom changes to make icons a bit larger too.

image
image
image

Additionally, collapses examine into a single to_chat by gathering a list of examine texts and displaying it in one go.

## Why It's Good For The Game

It makes examining things way easier.

## Changelog

:cl:
qol: Things you examine are now contained in a block, making it easier to see. Same applies to checking yourself for injuries.
/:cl:

### CODE REVIEW

**Comment by morrowwolf** [comment]
Given the expansiveness of this change we should likely testmerge for any issues before fully merging.

**Comment by Tristan63** [comment]
There should be an option to disable it if you don't want the boxes.

File: code/game/objects/items/reagent_containers/pill.dm [hunk 1]
```diff
@@ -40,19 +40,20 @@
...
 
 /obj/item/reagent_container/pill/display_contents(mob/user)
 	if(isXeno(user))
 		return
 	if(!identificable)
 		return
 	if(skillcheck(user, SKILL_MEDICAL, SKILL_MEDICAL_TRAINED))
-		to_chat(user, pill_desc)
-	..()
+		return pill_desc
```
Comment (Shad0vvs) [inline code/game/objects/items/reagent_containers/pill.dm:56]:
```suggestion
		return "[src] contains: [get_reagent_list_text()]."//this the pill
```

Comment (Shad0vvs) [reply code/game/objects/items/reagent_containers/pill.dm:56]: Or make it work with parent cuz I couldnt :^)

**Comment by LoJoFan** [comment]
Examining custom pills no longer tells you their content. Examining default pills tells you the type, but not the content (unsure if this is for all people or just for those with medical skill).

**Comment by Shad0vvs** [comment]
> Examining custom pills no longer tells you their content. Examining default pills tells you the type, but not the content (unsure if this is for all people or just for those with medical skill).

Fixed above.

**Comment by Wintermote** [comment]
I found these boxes to be very helpful, I can't really describe why, but it definitely feels easier to read

**Comment by carlarctg** [comment]
soul change +1

**Review by Shad0vvs — APPROVED** [review]
ok

**Pull request merged**